src/pudl/metadata/resources/ferc1.py

```diff
@@ -4,23 +4,28 @@
 from pudl.metadata.codes import CODE_METADATA
 
 RESOURCE_METADATA: dict[str, dict[str, Any]] = {
-    "accumulated_depreciation_ferc1": {
+    "accumulated_depreciation_plant_ferc1": {
         "description": "Balances and changes to FERC Accumulated Provision for Depreciation.",
         "schema": {
             "fields": [
                 "utility_id_ferc1",
                 "report_year",
-                "record_id",
-                "line_id",
-                "total",
+                "ferc_account_label",
+                "electric_plant_classification",
+                "utility_type",
                 "electric_plant",
```

Comment:
I really think we should not propagate a ton of different column names that all refer to $ values. we have several instances of `{something}_value` which imo is better than this name. or `balance`, which i think would be appropriate in this table? or just `value`!

Reply:
I decided to use `utility_plant_value` here since that's what was used in the other depreciation table (`utility_plant_summary_ferc1`) that seems to be referring to the same value (in fact I think this table is aggregated up into that one)